docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -342,7 +342,14 @@ The ``query`` function allows you to query the underlying database directly. It
 requires syntax native to PostgreSQL, because the full query is pushed down and
 processed in PostgreSQL. This can be useful for accessing native features which
 are not available in Trino or for improving query performance in situations
-where running a query natively may be faster.
+where running a query natively may be fasters.
+
+.. include:: query-passthrough-warning.fragment
+
+.. Warning::
+
+  Using this method allows any query that reads data to bypass access control,
+  as no validation or security checks are performed.
```

Comment:
It might be better to say that only security checks defined in the source system apply?

i.e. permissions from Trino won't apply but any permissions you have configured on the source system still apply.

Reply:
Yeah ... the draft I had for @Jessie212 had that. Still working out the final wording.

Reply:
@mosabua I'm just going to use your original draft, but with slight changes to the last paragraph.